<?xml version="1.0"?>

<launch>

  <arg name="sensor_id" default="imu_1" />
  <arg name="sensor_number" default="1" />

  <node pkg="imu_ros" type="imu_for_fusion" respawn="false" name="imu_detector" output="screen">
    <param name="sensor_id" type="string" value="$(arg sensor_id)"/>
    <param name="sensor_number" type="int" value="$(arg sensor_number)"/>
  </node>
</launch>
